operations/validate_csm_health.md

```diff
@@ -320,33 +320,64 @@ These tests may be executed on any one worker or master NCN (but **not** `ncn-m0
 
 Run the HMS CT tests. This is done by running the `run_hms_ct_tests.sh` script:
 
-```bash
+```text
```

Comment:
Isn't this ran in bash?

Reply:
Some of these are commands that run in the terminal but aren't technically valid BASH, like the <service> part of:
`/opt/cray/csm/scripts/hms_verification/run_hms_ct_tests.sh -t <service>`

From what I can tell the only difference in the code blocks between `text` and `bash` are that colorations are attempted to be set when using `bash` and they get set incorrectly or in a potentially confusing manner in some cases. I opted to use `text` since the colorations seem unnecessary and might do more harm than good in these examples.

Reply:
That being said, it's just my preference so if we are aiming to make this standardized across the documentation I can change them to `bash`.